package.json

```diff
@@ -5,51 +5,78 @@
   "license": "MIT",
   "babel": {
     "presets": [
-      "@dosomething/babel-config"
+      [
+        "@dosomething",
+        {
+          "withGraphQL": false,
+          "withPolyfills": false
+        }
+      ]
     ]
   },
+  "browserslist": [
+    ">0.5%",
+    "ie 11",
+    "not op_mini all"
+  ],
   "eslintConfig": {
     "extends": "@dosomething/eslint-config",
     "globals": {
       "document": false,
       "window": false
     }
   },
+  "prettier": {
+    "singleQuote": true,
+    "trailingComma": "all"
+  },
   "main": "lib/index.js",
   "scripts": {
-    "prepublish": "npm run build",
-    "clean": "rm -rf dist",
-    "start": "npm run styleguide & npm run build:dev",
+    "start": "npm run styleguide & npm run modernizr && webpack --env=development --hide-modules --watch",
     "styleguide": "NODE_ENV=test babel-node styleguide/app.js",
-    "build": "npm run clean && npm run modernizr && NODE_ENV=production webpack && NODE_ENV=test babel js --out-dir lib",
-    "build:dev": "npm run clean && npm run modernizr && webpack",
+    "build": "webpack --env=production && NODE_ENV=test babel js --out-dir lib && npm run modernizr",
+    "build:dev": "npm run modernizr && webpack --env=development --hide-modules",
     "modernizr": "modernizr -c modernizr.json -d dist/modernizr.js",
-    "test": "eslint js"
+    "test": "eslint js",
+    "prepublish": "npm run build"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged"
+    }
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/DoSomething/forge.git"
   },
   "dependencies": {
-    "jquery": "^3.0.0",
+    "jquery": "^3.4.1",
```

Comment:
forgot we're still using this! haha

Reply:
Haha, I know! I'd love to remove this dependency, maybe during our cleanup sprint!